public/js/services/routingMiddleware.js

```diff
@@ -0,0 +1,40 @@
+import { replace } from 'react-router-redux';
+import { Actions } from 'jumpstate';
+import _isEqual from 'lodash/isEqual';
+import moment from 'moment';
```

Comment:
is `moment`needed? [dateFns](https://date-fns.org/docs/Getting-Started) can do a lot of the same but is far smaller and supports single member imports so we can take advantage of webpack's tree-shaking.

Reply:
So far we've been managing all dates on the app with moment objects, so this is the format they currently need to be in the state, but it seems sensible to switch to dateFns all across the app if it does everything that we use moment for! I would do that on a separate pr though.